analyzer/CMakeLists.txt

```diff
@@ -6,7 +6,6 @@ add_subdirectory(dhcp)
 add_subdirectory(dns)
 add_subdirectory(facefish_rootkit)
 add_subdirectory(http)
-add_subdirectory(ipsec)
```

Comment:
Just piggy-backing on this change which looks good:

Let's instead pull this package in via a dependency in `zkg.meta`. Please add the dependency with a specific version tag, e.g.,

```
depends = zeek-spicy-ipsec ==0.2.1
```

That way users of `spicy-analyzers` keep seeing the same analyzers.

Reply:
Done.  Please double check me.  Thanks!